bluecompute-mp/requirements.yaml

```diff
@@ -1,4 +1,4 @@
 dependencies:
```

Comment:
Please validate that this file is necessary in this context, as I believe `requirements.yaml` files only work inside the context of a Chart and not in a Chart repository.